docs/docs/creating-a-sitemap.md

```diff
@@ -2,7 +2,29 @@
 title: Creating a sitemap
 ---
 
-This is a stub. Help our community expand it.
+### What is a Site Map?
 
-Please use the [Gatsby Style Guide](/docs/gatsby-style-guide/) to ensure your
-pull request gets accepted.
+An [XML sitemap](https://support.google.com/webmasters/answer/156184?hl=en) lists a website’s important pages, making sure search engines (such as Google) can find and crawl them all, and helping it understand your website structure.
+
+Think of it as a map for your website. It shows what all of the pages are on your website.
+
+### Install
+
+To generate an XML sitemap, you will use the [`gatsby-plugin-sitemap`](/packages/gatsby-plugin-sitemap/) package. To install this package, run the following line on your command line:
+`npm install --save gatsby-plugin-sitemap`
+
+### How to use [gatsby-plugin-sitemap](/packages/gatsby-plugin-sitemap/)
+
+````javascript
```

Comment:
This code block is a little messed up. It's using a JavaScript fenced code block, but the contents inside are Markdown. I think you just need to remove the second level of backticks _and_ change to use 3 backticks instead of 4.

Reply:
Hi,
Could you clarify that step? I might be silly but I don't know what to do for that step 🤦‍♂️

Sorry!

Reply:
Something like this!

````markdown
```javascript:title=gatsby-config.js
module.exports = {
  siteMetadata: {
    siteUrl: `https://www.example.com`,
  },
  plugins: [`gatsby-plugin-sitemap`]
}
````

Reply:
Oh I gotcha now! Thank you so much!